examples/get-view-from-cube-demo.js

```diff
@@ -0,0 +1,31 @@
+const { Source, View } = require('..')
+
+async function main () {
+  const source = new Source({
+    endpointUrl: 'https://int.lindas.admin.ch/query'
+  })
+
+  const cube = await source.cube('https://ld.stadt-zuerich.ch/statistics/ZUS-BTA-ZSA')
```

Comment:
This file is a demo showing how to read a cube-view from a cube. It's not part of the lib, but an example